pycryptobot.py

```diff
@@ -827,6 +827,9 @@ def executeJob(sc, app=PyCryptoBot(), state=AppState(), trading_data=pd.DataFram
 
                     # execute a live market buy
                     state.last_buy_size = float(account.getBalance(app.getQuoteCurrency()))
+                    if app.getBuyMaxSize() and state.last_buy_size > app.getBuyMaxSize():
```

Comment:
I think here we should write 
```python
if app.getBuyMaxSize() is not None and state.last_buy_size > app.getBuyMaxSize():
```
I'm not a Python guy but I think its more robust.

Reply:
I think this is just another way to do the same thing

Reply:
I guess its a personal code style... I do prefer explicit code and avoid relying on type coercion when it's not require.

Reply:
It is indeed personal style. Both styles are used in this project. In think @whittlem  should decide which style he want for new code. I don't mind changing it.

Reply:
FYI here an official recommendation
https://www.python.org/dev/peps/pep-0008/#programming-recommendations